### PR TITLE
feat(presets): Add replacements:xmldom-to-scoped

### PR DESCRIPTION
**Pull request by karfau**
## Changes

I added a replacement preset that will upgrade any versions of the packages `xmldom` and `xmldom-alpha` to `@xmldom/xmldom@v0.7.5`

## Context

Latest versions of `xmldom-alpha` and `xmldom` have security issues, the package is now published under `@xmldom/xmldom`.

https://github.com/xmldom/xmldom/issues/271

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/presets/internal/replacements.ts [hunk 1]
```diff
@@ -120,4 +121,16 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'xmldom-to-scoped': {
+    description:
+      'the xmldom package is now published as @xmldom/xmldom, see https://github.com/xmldom/xmldom/issues/271',
```
Comment (viceice) [inline lib/config/presets/internal/replacements.ts:126]: i don't think we need the issue link here
```suggestion
      'the xmldom package is now published as @xmldom/xmldom,',
```

Comment (karfau) [reply lib/config/presets/internal/replacements.ts:126]: Just a question: Is this text only apearing in the documentaiton or also in the update PRs?

Comment (viceice) [reply lib/config/presets/internal/replacements.ts:126]: only in the docs and renovate logs, and the cause for the package  rename doesn't matter for renovate.

**Comment by karfau** [comment]
@viceice Is there still anything I can do to support this PR getting merged?

**Review by viceice — APPROVED** [review]

**Pull request merged**